lib/ui/widget/keypad/key_pad_buttons.dart

```diff
@@ -97,28 +105,35 @@ class SingleKey extends StatelessWidget {
     // ignore: always_specify_types
     return ViewModelBuilder.reactive(
```

Comment:
I think this doesn't need to be reactive so change it to `nonReactive `instead except if there is a convincing idea to make it reactive.